### DISCUSSION TITLE
content/index.md is not showing

### DISCUSSION THREAD

**Discussion by NyllRE** [nuxt/content#1449] [Q&A] [answered]
when I start a project freshly new, the routes are broken. the `index.md` will never be shown and for some reason going to `localhost:3000` it always reroutes me to the `about.md` even though the url seems to not change. has this ever happened to anyone else? it's weird because it is a fresh new project, untouched code.

also something weird is that it shows in the terminal that I'm using nuxt rc.8 but in the package.json file it shows nuxt rc.4

I have followed the [official nuxt guide](https://content.nuxtjs.org/get-started) to start the project

node: v16.16.0
npm: 8.11.0
nuxt: 3.0.0@rc.4 ?? maybe rc.8 idk it shows both

**Comment by kjanusz** [comment, marked as answer]
I can confirm the "problem" you described. 

with 

```json
{
  "devDependencies": {
    "@nuxt/content": "^2.0.0",
    "nuxt": "^3.0.0-rc.4"
  }
}
```

and

```json
{
  "devDependencies": {
    "@nuxt/content": "^2.0.1",
    "nuxt": "^3.0.0-rc.8"
  }
}
```
My recommendation: add new index.vue in pages directory, example:

```ts
<template>
    <main>
        <ContentList path="/">
            <!-- ...default slot -->
            <template #default="{ list }">
                <div v-for="post in list" :key="post._path">
                    <h2>{{ post.title }}</h2>
                    <p>{{ post.description }}</p>
                    LINK:
                    <nuxt-link :to="post._path">{{ post.title }}</nuxt-link>
                </div>
            </template>
            <template #not-found>
                <p>No articles found. ERROR 404</p>
            </template>
        </ContentList>
        <br />
        <hr />
        <br />
        <nuxt-link to="/">HOME</nuxt-link>
    </main>
</template>
```

**Comment by nobkd** [comment]
You can try using the [edge channel](https://content.nuxtjs.org/guide/migration/edge-channel) until v2.1.0 is released: #1439 
There are multiple issues describing the problem with the index routing (like #1237) that suggest to use the edge channel in the meantime.
Alternatively just add a `1.` to the front of your `index.md` to specify a [custom navigation order](https://content.nuxtjs.org/guide/writing/content-directory#ordering).

For the "problem" with the nuxt version:
I think your project uses the nuxt version `rc.4` but nuxi (the nuxt CLI) version `rc.8`. But that's just my hypothesis 🤷.
You could also upgrade your nuxt version by using `nuxi upgrade` in the root of your repo, or by changing the version manually in the `package.json` and installing the packages with your package manager again.
